### PR TITLE
Improve Accuracy of the AI model

### PR DESCRIPTION
**Pull request by Andrewed95**

### CODE REVIEW

File: purchase_invoice.py [hunk 1]
```diff
@@ -948,29 +957,42 @@ def set_items(self, invoice_json, given_supplier):
         self.order_id = get_element_with_high_confidence(invoice_json, 'order_id')
         self.gross_total = get_element_with_high_confidence(invoice_json, 'total_amount')
         if self.gross_total:
-            self.gross_total = self.gross_total.replace('USD', '')
             self.gross_total = get_float_number(self.gross_total)
+            if self.supplier and "Frappe Technologies" in self.supplier:
+                self.gross_total = round(self.gross_total * 0.87408, 2)
```
Comment (tillmo) [inline purchase_invoice.py]: where does the value 0.87408 come from? This seems to be overfitted.

**Review by tillmo — CHANGES_REQUESTED** [review]

File: purchase_invoice.py (hunk 1 above)
Comment (Andrewed95) [reply purchase_invoice.py]: Yes, the reason is that the price unit in the invoices of this supplier is not in EUR, and is in USD. Is there a better solution for that?

Comment (tillmo) [reply purchase_invoice.py]: Maybe there is, but we will not need this at the moment. Hence, please remove the code mentioning "Frappe Technologies", and remove the json1 values in the Frappe PreRechnungs, such that they will no longer be in the test set.

Comment (Andrewed95) [reply purchase_invoice.py]: Done

**Review by tillmo — APPROVED** [review]

**Pull request merged**